src/lib/components/ManagePanel.js

```diff
@@ -50,7 +50,7 @@ export default class ManagePanel extends React.Component {
     var parser = document.createElement('a');
     parser.href = url;
     if (parser.protocol.startsWith('http')) {
-      return parser.host;
+      return parser.host.replace(/^www\./, '');
     }
     return url;
```

Comment:
I wonder if we should just return the host here, and let the title distinguish the content…  I mean, it's not like we'll see much of the path anyways.